sdk/build-scripts/package.json

```diff
@@ -9,7 +9,8 @@
 		"node": ">=16"
 	},
 	"bin": {
-		"build-package": "./src/build-package.ts"
+		"build-package": "./src/build-package.ts",
+		"build-dapp-kit": "./src/build-dapp-kit.ts"
```

Comment:
It felt a _little_ weird to place the `dapp-kit` build script in `build-scripts` since the script is only for dapp-kit, but I had trouble configuring the package.json for dapp-kit to successfully run the script and then I thought maybe it isn't so weird after all? I'd love to hear your thoughts on this @hayes-mysten!

Reply:
I'm fine with this 🤷, scripts are there to make stuff easier, it's not like we are going to publish/share these other places...